Mathlib/RingTheory/MvPolynomial/Symmetric.lean

```diff
@@ -174,24 +183,61 @@ def renameSymmetricSubalgebra [CommSemiring R] (e : σ ≃ τ) :
     (AlgHom.ext <| fun p => Subtype.ext <| by simp)
     (AlgHom.ext <| fun p => Subtype.ext <| by simp)
 
+variable (σ R : Type*) [CommSemiring R] [CommSemiring S] [Fintype σ] [Fintype τ]
+
+section Partitions
+
+/-! ### Multiplicativity on partitions -/
+
+variable (f : ℕ → MvPolynomial σ R)
+
+/-- Given a sequence of `MvPolynomial` functions `f` and a partition `μ` of size `n`,
+`muProduct` computes the product of applying each function in `f` to the parts of `μ`. -/
+def muProduct {n : ℕ} (μ : n.Partition) : MvPolynomial σ R :=
+  (μ.parts.map f).prod
```

Comment:
I think you should inline this definition. It's very easy to simply write `(μ.parts.map f).prod`